app/src/main/java/org/oppia/app/topic/review/ReviewAdapter.kt

```diff
@@ -7,42 +7,42 @@ import androidx.databinding.library.baseAdapters.BR
 import androidx.recyclerview.widget.RecyclerView
 import org.oppia.app.R
 import org.oppia.app.databinding.TopicReviewSummaryViewBinding
-import org.oppia.app.model.SkillSummary
+import org.oppia.app.model.Subtopic
 
 // TODO(#216): Make use of generic data-binding-enabled RecyclerView adapter.
 /** Adapter to bind skills to [RecyclerView] inside [TopicReviewFragment]. */
-class ReviewSkillSelectionAdapter(private val reviewSkillSelector: ReviewSkillSelector) :
-  RecyclerView.Adapter<ReviewSkillSelectionAdapter.SkillViewHolder>() {
+class ReviewAdapter(private val reviewSelector: ReviewSelector) :
+  RecyclerView.Adapter<ReviewAdapter.SkillViewHolder>() {
 
-  private var skillList: List<SkillSummary> = ArrayList()
+  private var subtopicList: List<Subtopic> = ArrayList()
 
   override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): SkillViewHolder {
-    val skillListItemBinding = DataBindingUtil.inflate<TopicReviewSummaryViewBinding>(
+    val reviewListItemBinding = DataBindingUtil.inflate<TopicReviewSummaryViewBinding>(
       LayoutInflater.from(parent.context),
       R.layout.topic_review_summary_view, parent,
       /* attachToRoot= */ false
     )
-    return SkillViewHolder(skillListItemBinding)
+    return SkillViewHolder(reviewListItemBinding)
   }
 
   override fun onBindViewHolder(skillViewHolder: SkillViewHolder, i: Int) {
-    skillViewHolder.bind(skillList[i], i)
+    skillViewHolder.bind(subtopicList[i], i)
   }
 
   override fun getItemCount(): Int {
-    return skillList.size
+    return subtopicList.size
   }
 
-  fun setSkillList(skillList: List<SkillSummary>) {
-    this.skillList = skillList
+  fun setReviewList(subtopicList: List<Subtopic>) {
+    this.subtopicList = subtopicList
     notifyDataSetChanged()
   }
 
   inner class SkillViewHolder(val binding: TopicReviewSummaryViewBinding) : RecyclerView.ViewHolder(binding.root) {
-    internal fun bind(skill: SkillSummary, @Suppress("UNUSED_PARAMETER") position: Int) {
-      binding.setVariable(BR.skill, skill)
+    internal fun bind(subtopic: Subtopic, @Suppress("UNUSED_PARAMETER") position: Int) {
```

Comment:
Remove this unused parameter. I think we won't be needing it.

Reply:
If not used please removed this.

Reply:
Removed.